README.md

```diff
@@ -14,6 +15,18 @@ python -m ipykernel install --user --display-name "Python (rapidgenomics)"
 
 After installing the necessary dependencies, you can just run `jupyter lab`.
 
+### Docker container
+Container with all dependency with notebook and source code are available at https://hub.docker.com/r/claraparabricks/single-cell-examples_rapids_cuda10.2-runtime-ubuntu18.04-py3.8.
```

Comment:
Change to 'A container with all dependencies, notebooks and source code is available at https://hub.docker.com/r/claraparabricks/single-cell-examples_rapids_cuda10.2-runtime-ubuntu18.04-py3.8.'

Also the link doesn't seem to work right now.

Reply:
Updated the changed container name along with link to hub.docker.com